compiler/luci/pass/src/QuantizeDequantizeWeightsPass.cpp

```diff
@@ -61,34 +61,45 @@ void compute_sym_scale_zp(float min, float max, float &scaling_factor, int64_t &
 void compute_asym_scale_zp(float min, float max, float &scaling_factor, int64_t &zp,
                            float &nudged_min, float &nudged_max)
 {
-  assert(min != max);
+  LOGGER(l);
 
+  assert(min <= max);
   const int32_t kMinScale = 0;
   const int32_t kMaxScale = 255;
   const double qmin_double = kMinScale;
   const double qmax_double = kMaxScale;
   const double rmin = std::fmin(0, min);
   const double rmax = std::fmax(0, max);
-
```

Comment:
Please split this change to another PR, maybe you can call it enhance readability.